### ISSUE TITLE
torch.randint和paddle.randint生成不对应

### ISSUE THREAD

**Issue by Yang-Changhui** [PaddlePaddle/Paddle#60293]
### bug描述 Describe the Bug

当使用torch.randint时
`import torch
seed = torch.randint(0, 2 ** 63 - 1, []).item()`,
结果为；4465045006717095580，
当使用paddle.randint时：
`import paddle
seed = paddle.randint(0, 2 ** 63 - 1, []).item()`,
结果报错：ValueError: (InvalidArgument) randint's low must less then high, but received: low = 0, high = -1.
  [Hint: Expected low < high, but received low:0 >= high:-1.] (at ../paddle/phi/infermeta/nullary.cc:112)

**Comment by zhiqiu** [comment]
感谢反馈。我查看了下，在paddle中该算子的输入Low和high类型为int，而torch用的是SymInt（一种自定义类型，可支持int64），这是一个非常好的建议，我们需要评估下类似支持 int64_t 的工作。

**Comment by jeff41404** [comment]
Indeed, as @zhiqiu mentioned, the value of `high` exceeded the range of int32, which caused the problem. I would also like to further confirm in which scenarios or models it is necessary to use

**Comment by Yang-Changhui** [comment]
我在复现modelscope video2video项目中，使用BrownianTreeNoiseSampler进行噪音采样，生成种子时，
https://github.com/modelscope/modelscope/blob/master/modelscope/models/multi_modal/video_to_video/utils/solvers_sdedit.py#L83
